### ISSUE TITLE
Add the soft mode of the redux effect

### ISSUE THREAD

**Issue by l16i** [flyingrub/Redux#1]
Hi @flyingrub,
Great work you did here!
I'm interested in this effect but did you find a way to recreate the "soft mode" of the Redux? I'm not sure to see it in your code.

**Comment by flyingrub** [comment]
The soft mode just limit the range of downsample parameter from what I saw

**Comment by l16i** [comment]
@flyingrub Yes it's one of its feature but there are an additional interpolation step that I cannot reproduce. It doesn't seems to be a simple linear interpolation 🤔.
If you test the soft and hard mode in the same range, you can hear a difference. The soft mode is more "clear".

**Comment by flyingrub** [comment]
Probably they use a different dry/wet settings. :)

**Comment by l16i** [comment]
Thanks for the input @flyingrub I will test it.
